checkmarx/project.py

```diff
@@ -285,3 +115,30 @@ def scan_finished():
 
     def scan_statistics(self, scan_id: int):
         return self.client.get_scan_statistics(scan_id=scan_id)
+
+    def get_last_scans(self):
+        return self.client.get_last_scans_of_project(project_id=self.project_details.id)
```

Comment:
hmmm.... somehow, this class seems to wrap the cx-client, with the main difference that it will implicitly pass-in the selected project-id?

if this is the case, I think it might be the better choice to rather add some convenience methods to the api-clients, or maybe just some util-funtions, that accept the client (+project-id) as parameter (thus requiring all relevant parameters to always be passed-in explicitly).

this has of course also the disadvantage of increased number of repetitive arguments, so this is just a thought - I am not 100% sure whether it would actually improve things overall.